third_party/proto/tendermint/abci/types.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package tendermint.abci;
 
-option go_package = "github.com/celestiaorg/celestia-core/abci/types";
+option go_package = "github.com/tendermint/tendermint/abci/types";
```

Comment:
this closes [#145](https://github.com/celestiaorg/celestia-core/issues/145)! :raised_hands: This is actually really nice, as we will be able to use the cli in scripts w/o getting messed up output. i.e 
```sh
($celestia-appd tendermint show-node-id)
```